test_integration/02_stake_and_mint/utils/progress_mint_assertion.js

```diff
@@ -218,14 +221,22 @@ class ProgressMintAssertion {
     );
     assert.strictEqual(
       eventData._proofProgress,
-      false,
+      proofProgress,
       'Proof progress flag should be false.',
     );
-    assert.strictEqual(
-      eventData._unlockSecret,
-      stakeRequest.unlockSecret,
-      'Unlock secret must match.',
-    );
+    if (proofProgress === true) {
```

Comment:
Didn't you assert in the lines above that it was false? 😕 I think the message in the assert above is wrong.

Reply:
Thanks 👍 
I corrected the assert messages for both asserts.